### PR TITLE
intro.adoc: suggested changes

### PR DESCRIPTION
**Pull request by grahamperrin**
Late changes, so feel to reject all suggestions. 

This pull request dovetails with the conversation at <https://github.com/freebsd/freebsd-quarterly/pull/535#pullrequestreview-1143200832> about possible methods of contacting the team, or individual members. 

Link to the main FreeBSD Status Reports page, which is not directly available through the current sidebar (it's indirectly through the News link). 

Whilst here: maybe also link to last quarter's report. 

Without something (first) 'on one hand', there should be nothing 'on the other hand'. 

Avoid combining 'maybe' with 'might'. 

The phrase 'quarterly team' is a little ambiguous. Better: 'status report team' (or 'quarterly status report team') in the sign-off.

### CODE REVIEW

**Comment by grahamperrin** [comment]
> … possible methods of contacting the team, or individual members.
> 
> Link to the main FreeBSD Status Reports page, …

### For clarity

[FreeBSD Status Reports](https://www.freebsd.org/status/): 

* already offers the quarterly-submissions@ e-mail address
* might be improved by defining the team; and/or offering alternative methods of communication.

For what it's worth: whilst the roles of people such as @lsalvadore seem clear enough (from an insider's perspective), I don't imagine a _strict_ definition, of the type that's seen at [FreeBSD Project Administration and Management](https://www.freebsd.org/administration/) …

**Comment by lsalvadore** [comment]
I think the introduction can stay as it is, I prefer not to touch the report anymore after publication except for typos, mistakes and things that are clearly wrong. However, let's discuss your suggestions anyway so we do better next time.

> Link to the main FreeBSD Status Reports page, which is not directly available through the current sidebar (it's indirectly through the News link).

That is an excellent idea: quarterly status reports definitely need to be easier to find. I do not thing adding a link in the signature of the introduction is the right strategy. I would use the current sidebar, or a link could be added to the top menu (maybe under Community).

> Without something (first) 'on one hand', there should be nothing 'on the other hand'.

I think this is about style, not about grammar, isn't it? I have done only a quick research on the web, but it seems to me that using 'on the other hand' without explicit 'on one hand' is allowed in English.

> Avoid combining 'maybe' with 'might'.

This is probably style again. I agree, using both was too much.

> The phrase 'quarterly team' is a little ambiguous. Better: 'status report team' (or 'quarterly status report team') in the sign-off.

> For what it's worth: whilst the roles of people such as @lsalvadore seem clear enough (from an insider's perspective), I don't imagine a strict definition, of the type that's seen at [FreeBSD Project Administration and Management](https://www.freebsd.org/administration/) …

Yes, the team name is always very ambiguous. Actually, if you look in /etc/aliases on freefall you will find that things are even more complex: the team is first defined as 'monthly' and refers to bi-monthly status reports. I am going to leave it as it is for now, but I am also going to try to make some more clarity on the topic.

**Comment by grahamperrin** [comment]
Thanks, close this PR whenever you like – anything actionable/substantial can be spun off to an issue or mailing list. 

----

> I would use the current sidebar, or a link could be added to the top menu (maybe under Community).

Via @sergio-carlavilla <https://www.freebsd.org/status/report-2022-07-2022-09/#_freebsd_manual_pages_portal>, **food for thought** (currently a _preview_ of changes to the portal for manual pages): <https://www.carlavilla.es/> 

* freedom from sidebars
* four menus, none of which is cluttered
* a footer that is similarly not too busy. 

The footer (preview), in essence: 

image

Be bold. 

The status of the FreeBSD Project encompasses the **truest reflections** of what the project is about, recently, so: 

* quarterly status reports belong under **About**

image

Then, move some of the best of <https://www.freebsd.org/status/> to <https://www.freebsd.org/about/>, and play with words in both pages until the upfront parts become: 

* **entirely positive and powerful**. 

For example: some people might like to read about a background that included (or includes)  _difficulties_ and _problems_, however these downsides need not be mentioned so near the head of the page.

----

Be bold! 

This is a definitely a discussion for another area ([freebsd-advocacy](https://lists.freebsd.org/archives/freebsd-advocacy/), maybe) and another day, but here goes: 

* the longer we have collective pussyfooting, the lower the likelihood of the Project gaining significant homepage-newsworthy press coverage ahead of its thirtieth anniversary. 

It's neat that this year has brought us two Liam Proven articles in The Register (the two that are linked from the FreeBSD homepage), but then check this out: 

* [liam_on_linux | A fresh look at FreeBSD](https://liam-on-linux.dreamwidth.org/86277.html) 

– via [Valuable News – 2022/09/12 | 𝚟𝚎𝚛𝚖𝚊𝚍𝚎𝚗](https://vermaden.wordpress.com/2022/09/12/valuable-news-2022-09-12/) and <https://old.reddit.com/r/freebsd/comments/xg0w7v/-/>

----

Bottom line, here: I think of **Project status reports** plus **Foundation content** (the technology roadmap, in particular) as the most valuable combination of resources, when points of reference are needed to reshape a person's misunderstandings of what's going on. 

HTH

**Comment by grahamperrin** [comment]
> … the team name is always very ambiguous. …

Is `quarterly@` synonymous with `quarterly-submisions@`? 

(An inconsistency between `/status/howto/` and `/status/`.)

> … if you look in /etc/aliases on freefall …

I'll add to my to-do list. Thanks.

**Comment by lsalvadore** [comment]
Thank you very much @grahamperrin for your feedback. Your thoughts about where quarterly status reports should be appear are very interesting and I agree with you.

> It's neat that this year has brought us two Liam Proven articles in The Register (the two that are linked from the FreeBSD homepage), but then check this out:
>
> [liam_on_linux | A fresh look at FreeBSD](https://liam-on-linux.dreamwidth.org/86277.html)

It is true that FreeBSD could be much more user friendly. I like it as it is, I would even say that I like it more because it is not too much user friendly (user friendliness comes at a cost), but still I think we should do better about it. Indeed, I have found myself recommending Linux instead of FreeBSD because of this issue. I have some ideas to improve things, but they are very complex and will require much time, and it might be too much for work, so no spoiler for something that might never be ready.

> Also, is quarterly@ synonymous with quarterly-submisions@?

Yes, the set of recipients of quarterly-submissions@ is defined as identical to the set of recipients of quarterly@. You can see quarterly-submissions@ as an email alias for quarterly specifically thought for submitting reports, while quarterly@ is used for communication with or within the team.
